tests/kernel/poll/src/test_poll.c

```diff
@@ -253,13 +253,48 @@ void check_results(struct k_poll_event *events, uint32_t event_type,
 /**
  * @brief Test polling with wait
  *
+ * @ingroup kernel_poll_tests
+ *
  * @details
+ * Test Objective:
  * - Test the poll operation which enables waiting concurrently
  * for one/two/all conditions to be fulfilled.
  * - set a single timeout argument indicating
  * the maximum amount of time a thread shall wait.
  *
- * @ingroup kernel_poll_tests
+ * Testing techniques:
+ * - function and block box testing.
+ * - Interface testing.
+ * - Dynamic analysis and testing.
+ *
+ * Prerequisite Conditions:
+ * - CONFIG_TEST_USERSPACE
+ * - CONFIG_DYNAMIC_OBJECTS
+ * - CONFIG_POLL
+ *
+ * Input Specifications:
+ * - N/A
+ *
+ * Test Procedure:
+ * -# Use FIFO/semaphore/signal object to define poll event.
+ * -# Initialize the FIFO/semaphore/signal object.
+ * -# Create a sub thread to put FIFO,
+ * give semaphore and raise signal.
+ * -# Check the result when signal be raised, semaphore be given
```

Comment:
It better to change it with:

"Check the result when signal is raised, semaphore is givn and fifo is filled."

Also, plese check line 285 ~ 288.

Reply:
fixed,  thanks Steven.